js/externals.js

```diff
@@ -26,7 +26,7 @@ module.exports = () => ({
   'redux-thunk': 'ReduxThunk',
   redux: 'Redux',
   config: 'Config',
-  url: 'url',
+  'node-url': 'NodeUrl',
```

Comment:
```suggestion
  'url': 'NodeUrl',
```

I think we need to keep this to `url`. Basically, the left side is the package name you'll need to use in your import statement, while the right side is the key it will saved under on the `window` object. If we use `node-url`, I think we'll have to change all our import statement to be `import * from "node-url";`

I tested this locally with `yarn link`. I rebuilt admin and vesioned-admin. With `'node-url': 'NodeUrl'`, versioned-admin seemed to work, but it was bundling `url`. After I changed it to `'node-url': 'NodeUrl'`, it was still working, but `url` wasn't being bundled.

Reply:
Right, and `url` is safe at this stage because it's not in the `window` context. Will tweak.